app/main.py

```diff
@@ -1,6 +1,30 @@
-# write your imports here
+from app.cinema.bar import CinemaBar
+from app.people.cinema_staff import Cleaner
+from app.cinema.hall import CinemaHall
+from app.people.customer import Customer
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(
+        customers: list,
+        hall_number: int,
+        cleaner: str,
+        movie: str) -> None:
+    customer_instances = [
+        Customer(customer["name"], customer["food"])
+        for customer in customers
+    ]
+    new_cinema_hall = CinemaHall(hall_number)
+    new_cinema_bar = CinemaBar()
+    new_cleaner = Cleaner(cleaner)
```

Comment:
The `new_` part is not adding any value to the understanding its content, that is why this part could be omitted:
```suggestion
    cinema_hall = CinemaHall(hall_number)
    cinema_bar = CinemaBar()
    cleaner_person = Cleaner(cleaner)
```